python/ray/llm/tests/batch/cpu/processor/test_http_request_proc.py

```diff
@@ -21,3 +25,7 @@ def test_http_request_processor():
         "Authorization": "Bearer 1234567890"
     }
     assert stage.fn_constructor_kwargs["qps"] == 2
+
+
+if __name__ == "__main__":
```

Comment:
is this new? why?

Reply:
I found that without this no tests will be ran 🫠

Reply:
Good catch :D